docs/UserGuide.md

```diff
@@ -297,43 +298,60 @@ Example of usage:
 
 Expected output:
 
+### <a id="tag-n"><ins>3.13 Tag/Untag Notes:</ins> `tag`</a>
+Tags or untags a note with the given tag name.
 
-### <a id="delete-t"><ins>3.13 Delete Tags:</ins> `delete-t`</a>
-Deletes a tag from the list of tags and remove the tag from the related notes.
+Format: `tag-n /i INDEX /tag TAG [TAG COLOR] [/tag TAG]...`
 
-Format: `delete-t /tag TAG [/tag TAG]...`
+- Tags a note with a given TAG. If the note already contains the TAG, the tag will be removed from the note.The index refers to the index number shown in the displayed note list. The index must be a **positive integer** (1, 2, 3, …).
+- The user can create tag or untag multiple tags to a note within a single line.
 
-- Deletes the tag with the name TAG.
-- The user can delete multiple tags within a single line.
-- Notes with the tag, will have the tag removed.
+💡 If the user tries to tag a non existing tag to a note, the application will automatically create the tag for it.
 
 Example of usage: 
 
-`delete-t /tag CS2113`
+`Tag-n /i 1 /tag CS2113`
+
+`Tag-n /i 1 /tag CS2113 /tag important`
 
 Expected output:
 
+### <a id="tag-e"><ins>3.14 Tag/Untag Events:</ins> `tag`</a>
+Tags or untags an event with the given tag name.
 
-### <a id="tag"><ins>3.14 Tag/Untag Notes:</ins> `tag`</a>
-Tags or untags a note with the given tag name.
+Format: `tag-e /i INDEX /tag TAG [TAG COLOR] [/tag TAG]...`
 
-Format: `tag /i INDEX /tag TAG [TAG COLOR] [/tag TAG]...`
-
-- Tags a note with a given TAG. If the note already contains the TAG, the tag will be removed from the note.The index refers to the index number shown in the displayed note list. The index must be a **positive integer** (1, 2, 3, …).
-- The user can create tag or untag multiple tags to a note within a single line.
+- Tags a note with a given TAG. If the event already contains the TAG, the tag will be removed from the event.The index
+ refers to the index number shown in the displayed event list. The index must be a **positive integer** (1, 2, 3, …).
+- The user can create tag or untag multiple tags to an event within a single line.
 
-💡 If the user tries to tag a non existing tag to a note, the application will automatically create the tag for it.
+💡 If the user tries to tag a non existing tag to an event, the application will automatically create the tag for it.
 
 Example of usage: 
 
-`Tag /i 1 /tag CS2113`
+`Tag-e /i 1 /tag CS2113`
+
+`Tag-e /i 1 /tag CS2113 /tag important`
+
+Expected output:
+
+### <a id="delete-t"><ins>3.15 Delete Tags:</ins> `delete-t`</a>
+Deletes a tag from the list of tags and remove the tag from the related notes.
```

Comment:
Only tags? Can we do it for events as well?

Reply:
This is deleting of the tag and not untagging. Will update the content.